packages/settings-library/tests/test_postgres.py

```diff
@@ -17,15 +17,12 @@ def test_cached_property_dsn(mock_environment: dict):
     settings = PostgresSettings()
 
     # all are upper-case
-    assert all(key == key.upper() for key in settings.dict())
-
-    # dsn is computed from the other fields
-    assert "dsn" not in settings.dict()
-
-    # causes cached property to be computed and stored on the instance
```

Comment:
In Pydantic v2, it is possible to annotate `@computed_fields`. It must be specified in advance if we want to include it in the model serialization. By default `@computed_fields(repr=True)` for public fields, `@computed_fields(repr=False)` for private ones.

See: https://docs.pydantic.dev/2.0/usage/computed_fields/